kedro/contrib/io/bioinformatics/sequence_dataset.py

```diff
@@ -95,18 +96,7 @@ def __init__(
 
         """
         self._filepath = filepath
-        default_load_args = {}  # type: Dict[str, Any]
-        default_save_args = {}  # type: Dict[str, Any]
-        self._load_args = (
-            {**default_load_args, **load_args}
-            if load_args is not None
-            else default_load_args
-        )
-        self._save_args = (
-            {**default_save_args, **save_args}
-            if save_args is not None
-            else default_save_args
-        )
+        super().__init__(load_args, save_args)
```

Comment:
I prefer calling `super` at the top of the constructor, so the subclass would overwrite stuff from the parent, as a "specialisation" of the superclass.

Reply:
> I prefer calling `super` at the top of the constructor, so the subclass would overwrite stuff from the parent, as a "specialisation" of the superclass.

Fair argument. I just left it in the same place where default arguments were previously handled (as close to the original as I could), but that makes sense.